index.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8"/>
+        <meta http-equiv="X-UA-Compatible" content="IE=edge"/>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+        <link rel="stylesheet" href="index.css"/>
+        <title>Elite Library</title>
+    </head>
+    <body>
+        <section class="main-container">
+        <nav class="main-nav">
+            <div class="logo">Awesome Books</div>
+            <ul class="list-items menu">
+                <li class="book-list"><a class="list-link list-menu" href="">List</a></li>
+                <li class="add-book"><a class="add-link add-new" href="">Add new</a></li>
+                <li class="contact-list"><a class="contact-link contact-menu" href="">Contact</a></li>
+            </ul>
+        </nav>
+
+      <div class="date">
+        <p class="time"></p>
+      </div>
+
+        <div class="container">
+            <div class="all">
+                <h1>Awesome Books</h1> <br>
+
+                <table class="table booksContainer">
+                    <tbody id="bookList">
+
+                    </tbody>
+                </table>
+            </div>
+            <form id="enter">
+                <h1>Add books here</h1>
+                <input type="text" id="title" placeholder="Title"/></br>
+                <input type="text" id="author" placeholder="Author"/></br>
+                <div id=submit>
+                    <button id="submit" class="btn" type="submit">Add</button>
```

Comment:
According to the wireframe, the button `add` should be set to the right, look at the screenshot below;

<img width="263" alt="image" src="https://user-images.githubusercontent.com/31897434/216021181-8cd55937-82e4-4117-80b9-b2cf02ef1749.png">


Because this is the only issue with this PR, I would go ahead and approve it, please always pay close attention to details.

Reply:
Thank you for the review, I looked into that and fixed the positioning